category/traversable/lemmas.lean

```diff
@@ -0,0 +1,70 @@
+
+import tactic.cache
+import category.traversable.basic
+
+universe variables u
+
+open is_lawful_traversable
+open function (hiding comp)
+open functor
+
+lemma map_id_mk {α β : Type u}
+  (f : α → β) :
+  map f ∘ @id.mk α = @id.mk β ∘ f :=
+rfl
+
+attribute [functor_norm] is_lawful_traversable.naturality
+
+section traversable
```

Comment:
This should be a namespace; some of the theorems like `purity` seem a bit too vague to go in the root namespace.